
public class Exception {

	public static void main(String[] args) {

		System.out.println("프로그램 시");
		int value = 0;
		int num = 15/value;
		int result = num + 100;
		System.out.println("연산된 값 : " + result );
		System.out.println("프로그램 종료 ");
	
	}

}
